experimental/rocm/dynamic_symbols.c

```diff
@@ -29,7 +29,7 @@ static iree_status_t iree_hal_rocm_dynamic_symbols_resolve_all(
         syms->loader_library, kName, (void **)&syms->rocmSymbolName)); \
   }
 #define RC_PFN_STR_DECL(rocmSymbolName, ...) RC_PFN_DECL(rocmSymbolName, ...)
-#include "experimental/rocm/dynamic_symbol_tables.h"
+#include "experimental/rocm/dynamic_symbol_tables.h"  // IWYU pragma: keep
```

Comment:
is the comment needed?

Reply:
Following the export tag patch [here](https://github.com/google/iree/commit/a0ee7730417a12475dc14508f58c5a25c8de8294#diff-4705a22fccec27fa54445479effa1c26467ae99d224a2102750d90a1b68bd74c)

Reply:
Ha missed that. Looks good.